lib/addons/jupyterhub/index.ts

```diff
@@ -0,0 +1,52 @@
+import { Construct } from "constructs";
+import { ClusterInfo } from '../../spi';
+import { createNamespace } from '../../utils';
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from '../helm-addon';
+
+/**
+ * Configuration options for the add-on.
+ */
+ type JupyterHubAddOnProps = HelmAddOnUserProps;
+
+const JUPYTERHUB = 'jupyterhub';
+const RELEASE = 'blueprints-addon-jupyterhub';
+
+/**
+ * Defaults options for the add-on
+ */
+const defaultProps: HelmAddOnProps = {
```

Comment:
Is there anything we can offer for user management other than admin/admin e.g. based on [this](https://zero-to-jupyterhub.readthedocs.io/en/latest/administrator/authentication.html).

Reply:
There is a cognito integration that maps as a pattern - was going to add for patterns as commented above.

Reply:
That works